lldb/include/lldb/Target/DynamicLoader.h

```diff
@@ -337,6 +338,17 @@ class DynamicLoader : public PluginInterface {
     return std::nullopt;
   }
 
+  /// Returns a list of memory ranges that should be saved in the core file,
+  /// specific for this dßynamic loader.
+  ///
+  /// By default, this returns an empty list, but for POSIX/ELF it will return
```

Comment:
Let's not refer to specific subclasses here. It's an indication of bad layering and is bound to get out of date if anyone else implements this method. You could say "For example, an implementation can return the memory ranges necessary to locate thread local data (via GetThreadLocalData) in the saved core file."